### PR TITLE
update tmux-resurrect and tmux-continuum

### PR DESCRIPTION
**Pull request by RonanMacF**
###### Motivation for this change
The plugins seemed a bit outdated so I have updated these two.

###### Things done
ran `nix-prefetch-url` with in the repo and updated the sha256. Updated rev with the commit ID of the most recent commit in the respective repor

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -70,8 +70,8 @@ in rec {
     src = fetchFromGitHub {
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:70]: :arrow_up: Could we update the `version` attribute to reflect the commit date of https://github.com/tmux-plugins/tmux-continuum/commit/26eb5ffce0b559d682b9f98c8d4b6c370ecb639b

File: pkgs/misc/tmux-plugins/default.nix [hunk 2]
```diff
@@ -270,8 +270,8 @@ in rec {
...
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:270]: :arrow_up: Same here with `version` attribute.

File: pkgs/misc/tmux-plugins/default.nix [hunk 3]
```diff
@@ -70,8 +70,8 @@ in rec {
...
       owner = "tmux-plugins";
       repo = "tmux-continuum";
-      rev = "1531b3770a7cf7373d15fedd239c5331b99342d1";
-      sha256 = "1w3f7gzvv1k25yfr6d1snr2z88p8f87cahrbaslmyphdxpy0fa4m";
+      rev = "26eb5ffce0b559d682b9f98c8d4b6c370ecb639b";
+      sha256 = "1z4zx15y0lbkrjapzszg8mhb86i5p3ark71vsl3a3pwb7cx8c3fa";
     };
     dependencies = [ resurrect ];
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:76]: :medal_sports: Bonus point to complete the `meta` section of this package ([example](https://github.com/nixos/nixpkgs/blob/39e62d5d6e26668a818fdbdae46c4f571447f4e5/pkgs/tools/misc/tmux/default.nix#L57-L79)).

Most importantly the following attributes
```
homepage
description
license
platforms
maintainers
```

File: pkgs/misc/tmux-plugins/default.nix [hunk 4]
```diff
@@ -270,8 +270,8 @@ in rec {
     src = fetchFromGitHub {
       owner = "tmux-plugins";
       repo = "tmux-resurrect";
-      rev = "327c0481ad20c429b4e692e092659f8b3346b08f";
-      sha256 = "0nxfqazww36wwv49dzd39kq4jfls20834hf1458sf5pvmv5cmbyw";
+      rev = "e4825055c92e54b0c6ec572afc9b6c4723aba6c8";
+      sha256 = "1bh59axhk7ixvdbl676haw5zp697m3008381hj9zv0c9snyffphf";
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:274]: Double bonus points for completing the meta here :medal_sports: :medal_sports:.

**Review by timstott — COMMENTED** [review]
Thanks for your contribution :bow:, I have added several comments I hope we can address.

Also could we split each plugin update into its own commits with the template message below (in accordance with [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md)):
```
tmuxPlugins.continuum: unstable-2018-02-23 -> xxx
```

File: pkgs/misc/tmux-plugins/default.nix (hunk 3 above)
Comment (RonanMacF) [reply pkgs/misc/tmux-plugins/default.nix:76]: tell me how they look :)

Comment (timstott) [reply pkgs/misc/tmux-plugins/default.nix:76]: Smashing :tada:

File: pkgs/misc/tmux-plugins/default.nix [hunk 5]
```diff
@@ -66,14 +66,32 @@ in rec {
...
+        * automatic restore when tmux is started
+
+        Together, these features enable uninterrupted tmux usage. No matter the
+        computer or server restarts, if the machine is on, tmux will be there how
+        you left it off the last time it was used.
+      '';
+      license = stdenv.lib.licenses.mit;
+      platforms = stdenv.lib.platforms.unix;
+      maintainers = with stdenv.lib.maintainers; [ ronanmacf ];
+    }
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:94]: Missing `;`

File: pkgs/misc/tmux-plugins/default.nix [hunk 6]
```diff
@@ -266,13 +284,39 @@ in rec {
...
+          * programs running within a pane! More details in the restoring programs doc.
+
+          Optional:
+          * restoring vim and neovim sessions
+          * restoring pane contents
+      '';
+      license = stdenv.lib.licenses.mit;
+      platforms = stdenv.lib.platforms.unix;
+      maintainers = with stdenv.lib.maintainers; [ ronanmacf ];
+    }
```
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:319]: Missing `;`

File: pkgs/misc/tmux-plugins/default.nix (hunk 5 above)
Comment (RonanMacF) [reply pkgs/misc/tmux-plugins/default.nix]: apologies, good to go!

**Comment by timstott** [comment]
This is looking better everyday. @RonanMacF. Today we need to fix the `sha256` for both plugins are they are invalid.

```
hash mismatch in fixed-output derivation '/nix/store/68jp895j4yxw94jyj0hvbvbv41qs9w59-source':
  wanted: sha256:1z4zx15y0lbkrjapzszg8mhb86i5p3ark71vsl3a3pwb7cx8c3fa
  got:    sha256:1glwa89bv2r92qz579a49prk3jf612cpd5hw46j4wfb35xhnj3ab
hash mismatch in fixed-output derivation '/nix/store/a75px1hp4y1h3aagcw0c03s9grjkamd9-source':
  wanted: sha256:1bh59axhk7ixvdbl676haw5zp697m3008381hj9zv0c9snyffphf
  got:    sha256:0a96drkx1kpadkbxabcnvb542p75xdh2dbizvlq2lac5ldpb4hmx
cannot build derivation '/nix/store/s3cij2kdl179c9267i2h7x2vlpayqkh1-tmuxplugin-resurrect-unstable-2020-09-18.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/kp0mi5kcjmcnlbcw87qdsba2xaplv2hg-tmuxplugin-continuum-unstable-2020-10-16.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/bgjqi89dmsq1ziki41av5njzgsk4ny9f-env.drv': 2 dependencies couldn't be built
[0 built (2 failed)]
error: build of '/nix/store/bgjqi89dmsq1ziki41av5njzgsk4ny9f-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/101971
2 packages failed to build:
tmuxPlugins.continuum tmuxPlugins.resurrect
```

You can obtain the `sha256`  for GitHub repos with the following command:

```shell
nix-shell -p nix-prefetch-github --command 'nix-prefetch-github tmux-plugins tmux-continuum --rev 26eb5ffce0b559d682b9f98c8d4b6c370ecb639b'
```

I would also encourage you review the PR locally with `nix-review pr 101971`

**Comment by RonanMacF** [comment]
Result of `nixpkgs-review pr 101971` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>tmuxPlugins.continuum</li>
    <li>tmuxPlugins.resurrect</li>
  </ul>
</details>

**Comment by RonanMacF** [comment]
> This is looking better everyday @RonanMacF. Today we need to fix the `sha256` for both plugins are they are invalid.
> 
> ```
> hash mismatch in fixed-output derivation '/nix/store/68jp895j4yxw94jyj0hvbvbv41qs9w59-source':
>   wanted: sha256:1z4zx15y0lbkrjapzszg8mhb86i5p3ark71vsl3a3pwb7cx8c3fa
>   got:    sha256:1glwa89bv2r92qz579a49prk3jf612cpd5hw46j4wfb35xhnj3ab
> hash mismatch in fixed-output derivation '/nix/store/a75px1hp4y1h3aagcw0c03s9grjkamd9-source':
>   wanted: sha256:1bh59axhk7ixvdbl676haw5zp697m3008381hj9zv0c9snyffphf
>   got:    sha256:0a96drkx1kpadkbxabcnvb542p75xdh2dbizvlq2lac5ldpb4hmx
> cannot build derivation '/nix/store/s3cij2kdl179c9267i2h7x2vlpayqkh1-tmuxplugin-resurrect-unstable-2020-09-18.drv': 1 dependencies couldn't be built
> cannot build derivation '/nix/store/kp0mi5kcjmcnlbcw87qdsba2xaplv2hg-tmuxplugin-continuum-unstable-2020-10-16.drv': 2 dependencies couldn't be built
> cannot build derivation '/nix/store/bgjqi89dmsq1ziki41av5njzgsk4ny9f-env.drv': 2 dependencies couldn't be built
> [0 built (2 failed)]
> error: build of '/nix/store/bgjqi89dmsq1ziki41av5njzgsk4ny9f-env.drv' failed
> https://github.com/NixOS/nixpkgs/pull/101971
> 2 packages failed to build:
> tmuxPlugins.continuum tmuxPlugins.resurrect
> ```
> 
> You can obtain the `sha256` for GitHub repos with the following command:
> 
> ```shell
> nix-shell -p nix-prefetch-github --command 'nix-prefetch-github tmux-plugins tmux-continuum --rev 26eb5ffce0b559d682b9f98c8d4b6c370ecb639b'
> ```
> 
> I would also encourage you review the PR locally with `nix-review pr 101971`

All done, sha update and have posted the result of nix-review.

**Review by timstott — APPROVED** [review]
Excellent! Thanks @RonanMacF

**Review by infinisil — APPROVED** [review]
Great review @timstott and nice PR @RonanMacF, looking good!

**Pull request merged**